### DISCUSSION TITLE
Please Add CodeBehind Framework

### DISCUSSION THREAD

**Discussion by elanatframework** [TechEmpower/FrameworkBenchmarks#9085] [General]
Please also add the CodeBehind back-end framework in performance tests.

CodeBehind  Framework GitHub link:
https://github.com/elanatframework/Code_behind

CodeBehind is a new framework whose first version was released on .NET Core 7.0 in 2023.

CodeBehind is owned by [Elanat](https://elanat.net) and is a powerful competitor to Microsoft's default ASP.NET Core web frameworks (Razor Pages and ASP.NET Core MVC).

According to the Elanat team's latest tests, the CodeBehind framework outperforms Razor Pages and ASP.NET Core MVC by up to 40%.

Performance test, ASP.NET Core MVC and Razor Pages vs CodeBehind Framework
https://github.com/elanatframework/Code_behind/blob/elanat_framework/doc/performance_test_asp_dot_net_core_mvc_and_razor_pages_vs_code_behind_framework_version_2.2.md

**Comment by joanhey** [comment]
This benchmark is open source, and anybody is welcome to add a new framework.
